conf/etci2021.yaml

```diff
@@ -7,7 +7,7 @@ experiment:
     encoder_weights: "imagenet"
     learning_rate: 1e-3
     learning_rate_schedule_patience: 6
-    in_channels: 7
+    in_channels: 6
```

Comment:
Previously we had 7 channel inputs because we concatenated a given water mask with the image inputs. Generating predictions with a water mask would be tricky at best / outside of the scope of torchgeo so I removed this.

Reply:
Wouldn't predictions be out_channels, not in_channels?

Reply:
Yes, but the number of in_channels changed from 7 to 6 because we were previously using a pre-computed water mask as a channel.